### ISSUE TITLE
do I need to do anything for GitHub actions?

### ISSUE THREAD

**Issue by nhooyr** [codecov/codecov-bash#169]
Hi, I'm running this inside GitHub actions.

Do I need to pass a token or set anything so it can figure out the CI?

**Comment by radekmie** [comment]
Hi @nhooyr. I guess this answer is no longer relevant to you, including the change of Actions syntax since March. I want to raise an issue that it's still **not** supported. I've successfully configured it using the `run` directive directly:

```yml
jobs:
  CI:
    # ...
    steps:
      # ...
      - name: Report coverage
        run: bash <(curl -s https://codecov.io/bash)
        env:
          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

The thing is that it _should_ work, but there was no release after GitHub Actions support got merged in #199 and fixed in #218. It reports the coverage anyway, but all of the reports count towards the `master` branch.

**Comment by nhooyr** [comment]
Going to close as it has been a while and I'm not sure if this is relevant anymore.
